docs/source/actions/custom-action.rst

```diff
@@ -13,3 +13,45 @@ the `run_action` method.
         def run_action(self):
             # Include here the logic of your action
             # (...)
+
+
+Fallback Actions
+================
+
+When creating your own custom actions, you can also add a fallback action to run if
+an action throws an unhandled exception. To do this, add a fallback attribute to
+your custom action.
+
+.. code-block:: python
+
+    from spidermon.core.actions import Action
+
+    class MyFallbackAction(Action):
+        def run_action(self):
+            # Include here the logic of your action
+            # Runs if MyCustomAction().run_action() throws an unhandled exception
+            # (...)
+
+    class MyCustomAction(Action):
+        fallback = MyFallbackAction
+        def run_action(self):
+            # Include here the logic of your action
+            # (...)
+
+
+You can also add fallbacks to spidermon built-in actions by subclassing them.
+
+.. code-block:: python
+
+    from spidermon.core.actions import Action
+    from spdiermon.contrib.actions import Sentry
+
+    class MyFallbackAction(Action):
+        # Runs if MyCustomSentryAction fails
+        def run_action(self):
+            # Include here the logic of your action
+            # (...)
+
+    class MyCustomSentryAction(Sentry):
+        fallback = MyFallbackAction
+
```

Comment:
Maybe we should add a real-world example like : 

- Action Sentry, Fallback Slack. 
- Action Upload to AWS, if fail Upload to GCS

Reply:
I added a email fallback to a slack action example now!